IOS/SwiftSeedProject/SwiftSeedProject/Infrastructure/PersistenceData.swift

```diff
@@ -20,54 +20,58 @@ class PersistenceData: Persistence {
     }
     
     public func getAll<T: PersistenceObject>(entityName: String) -> [T] {
-        let objectContext = coreDataManager.privateChildManagedObjectContext()
+        let objectContext = coreDataManager.mainManagedObjectContext
```

Comment:
why did you change the privateChildManagedObjectContext by MainManagedObjectContext?